index.ts

```diff
@@ -1291,22 +1293,74 @@ function getSyscalls() {
   };
 }
 
+function getModuleCache() {
+  return (global as any).Convex.modules as ReturnType<typeof moduleCache>;
+}
+
+function moduleCache(specifiedModules?: Record<string, () => Promise<any>>) {
+  const modules = specifiedModules ?? import.meta.glob("./convex/**/*.*s");
+  const prefix = findModulesRoot(
+    Object.keys(modules),
+    specifiedModules !== undefined,
+  );
+  const modulesWithoutExtension = Object.fromEntries(
+    Object.entries(modules).map(([path, module]) => [
+      path.replace(/\.[^.]+$/, ""),
+      module,
+    ]),
+  );
+  return async (path: string) => {
```

Comment:
nit: comment with an example usage would help me

seems like if I have `my-project/convex/foo.ts`, this'll map `my-project/convex/foo` -> `/abs/path/my-project/convex/foo` (prefix + path) -> the actual module?

Reply:
Yeah